inc/customizer.php

```diff
@@ -290,6 +290,24 @@ public static function register( $wp_customize ) {
 				'type'     => 'checkbox',
 			)
 		);
+		
+		// Show comment author's full name in reply-link
+		$wp_customize->add_setting(
+			'independent_publisher_general_options[show_comment_authors_full_name_in_reply_link]', array(
```

Comment:
I suggest changing the option key to `show_full_name_comment_reply_to` (this will need to be updated anywhere this key is used).